infrastructure/helm/quantumserverless/templates/pvcs.yaml

```diff
@@ -1,28 +1,21 @@
-apiVersion: v1
```

Comment:
I wonder if we can do something like this here:

```yaml
{{ if .Values.platform "cos" }}
kind: PersistentVolumeClaim
...
{{ end }}
{{ if eq .Values.platform "kind" }}
kind: PersistentVolumeClaim)     
...
{{ end }}
```

Reply:
Yeah, to try to introduce @akihikokuroda changes. What do you think about a logic like: `eq "kind",  not eq "kind"` (thinking that `platform: cos` sounds a bit weird?, maybe my impression). I'm assuming that in the cloud you are going to use `cos` and in local not.

Reply:
only issue there would be if you wanted to add a third/fourth/etc option (e.g. AWS)... maybe `platform: ibm`?

Reply:
You're right and I think that would work because probably configurations can be different between platforms too. Thanks @psschwei !

Reply:
@Tansito I think that the default case (not, kind or ibm) is missing.  Putting this just above the last {{ end }} seems good.
```
{{ else }}                                                                                                                                                                                                                                    
apiVersion: v1                                                                                                                                                                                                                                
kind: PersistentVolume                                                                                                                                                                                                                        
metadata:                                                                                                                                                                                                                                     
  name: gateway-volume                                                                                                                                                                                                                        
  labels:                                                                                                                                                                                                                                     
    type: local                                                                                                                                                                                                                               
  annotations:                                                                                                                                                                                                                                
    pv.beta.kubernetes.io/gid: "1001"                                                                                                                                                                                                         
spec:                                                                                                                                                                                                                                         
  storageClassName: manual                                                                                                                                                                                                                    
  capacity:                                                                                                                                                                                                                                   
    storage: 2Gi                                                                                                                                                                                                                              
  accessModes:                                                                                                                                                                                                                                
    - ReadWriteMany                                                                                                                                                                                                                           
  hostPath:                                                                                                                                                                                                                                   
    path: "/mnt/data/tmp"                                                                                                                                                                                                                     
---                                                                                                                                                                                                                                           
apiVersion: v1                                                                                                                                                                                                                                
kind: PersistentVolumeClaim                                                                                                                                                                                                                   
metadata:                                                                                                                                                                                                                                     
  name: gateway-claim                                                                                                                                                                                                                         
spec:                                                                                                                                                                                                                                         
  storageClassName: manual                                                                                                                                                                                                                    
  accessModes:                                                                                                                                                                                                                                
    - ReadWriteMany                                                                                                                                                                                                                           
  resources:                                                                                                                                                                                                                                  
    requests:                                                                                                                                                                                                                               
      storage: 1Gi  
```

Reply:
Thanks Aki, Iskandar. Solved in: [3e272e5](https://github.com/Qiskit-Extensions/quantum-serverless/pull/668/commits/3e272e52cc0c352ef6bf3bac0ced525f24ab84b0)